src/components/ContestTile.js

```diff
@@ -66,7 +68,7 @@ const ContestTile = ({ contest: { node } }) => {
         >
           {`${findingsRepo === "" ? "Preview" : "View"} Contest`}
         </Link>
-        {t.state === "active" && findingsRepo && submissionPath ? (
+        {t.state === "active" && findingsRepo && submissionPath && (end - now >= 0) ? (
```

Comment:
``` javascript
{t.state === "active" && findingsRepo && submissionPath && hasContestEnded() ? 
```

Reply:
Now that I'm looking at the rest of the code, I realized you can import `getTimeRemaining` and then check that the total is > 0

Reply:
So it would be
``` javascript
t.state === "active" && findingsRepo && submissionPath && getTimeRemaining().total > 0 ? 
```